.travis.yml

```diff
@@ -2,11 +2,7 @@ language: objective-c
 matrix:
   include:
     - osx_image: xcode11.6
```

Comment:
Why keep Xcode 11 around on Travis instead of moving everything to GH Actions? Also seems weird that Xcode 10 (older) would be on GH Actions, but Xcode 11 (newer) would be on Travis.

Reply:
Talked offline, this is because GitHub Actions doesn't support iOS 11. Derek will update the PR description to include this info.

Reply:
Updated